### PR TITLE
Tenant flag is now optional and other various changes

### PR DESCRIPTION
**Pull request by Nikokolas3270**

### CODE REVIEW

File: rule-repos/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- The [`Makefile`](./resources/Makefile) (used for local devlopment)
+- The [`README.md`](./resources/README.md)
+- The CI files.
+
+Pay attention to the comments in those files; make sure your proceed to the following adjustement when copying them:
+- Replace `<tenant>` with the RHOBS tenant targeted by the new repository.  
+  If the rule repository is targeting several tenants: replace `<tenant>` with the "base" tenant which is either one of the tenants or with some prefix common to all targeted tenants.
+- Adapt sections tagged with `<adapt-if-template>` regarding whether or not you will generate a template for the rules in your new rule repository.  
+  As a reminder, a template:
+  - **Agregates** all the Prometheus rules in a single object/file.
```
Comment (philipgough) [inline rule-repos/README.md:46]: typo: Aggregates

File: rule-repos/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  - **Is handy** in situations where you do not know in advance on what RHOBS tenant the rules will be deployed on.
+  - **Is required** when your rule repository targets several RHOBS tenant as the template let you define the targeted tenant when it is intanciated.
```
Comment (philipgough) [inline rule-repos/README.md:48]:
```suggestion
 - **Is required** when your rule repository targets several RHOBS tenants as the template lets you define the targeted tenant when it is instantiated.
```

File: rule-repos/resources/README.md [hunk 3]
```diff
@@ -1,81 +1,137 @@
...
+  - Remove this comment once done
+-->
+#   <tenant> RHOBS rules
+
+This repository contains the rules for the `<tenant>` tenant.
+
+<!--
+  TODO: Remove below sentence if the repository is not hosted on GitLab
+-->
+In GitLab, you can identifies the rules for the other RHOBS tenants [by filtering the repositories](https://gitlab.cee.redhat.com/explore/projects/topics/rhobs-rules) with the `rhobs-rules` topic.
```
Comment (philipgough) [inline rule-repos/resources/README.md:13]:
```suggestion
In GitLab, you can identify the rules for the other RHOBS tenants [by filtering the repositories](https://gitlab.cee.redhat.com/explore/projects/topics/rhobs-rules) with the `rhobs-rules` topic.
```

**Review by philipgough — APPROVED** [review]
some nits but lgtm. thanks

File: rule-repos/README.md (hunk 1 above)
Comment (Nikokolas3270) [reply rule-repos/README.md]: Thanks for pointing that out. Fixed.

File: rule-repos/resources/README.md (hunk 3 above)
Comment (Nikokolas3270) [reply rule-repos/resources/README.md]: Thanks for pointing that out. Fixed.

File: rule-repos/README.md (hunk 2 above)
Comment (Nikokolas3270) [reply rule-repos/README.md]: Thanks for pointing that out. Fixed.

**Pull request merged**